fedimintd/umbrel-app.yml

```diff
@@ -0,0 +1,66 @@
+manifestVersion: 1
+id: fedimintd
+category: bitcoin
+name: Fedimint
+version: "v0.8.0"
+tagline: Federated Chaumian E-Cash Mint for Bitcoin
+description: >-
+  Fedimint is a federated Chaumian E-Cash Mint to custody and transact bitcoin in a community. It allows groups of trusted individuals, 
+  called guardians, to jointly operate a multi-signature e-cash mint.
+
+  # Setting Up a Federation
+
+  You can create a Fedimint in two ways:
+
+    - Single Guardian Federation — A single guardian operating their own mint.
+
+    - Group Federation — Multiple guardians running the mint together (minimum of 4 guardians).
+
+  [See here for more deployment recommendations](https://github.com/fedimint/fedimint/blob/master/docs/deploying.md)
+
+  # Step By Step Ceremony
+
+  1. Install & Launch fedimintd
+     Each participating guardian runs their own instance of fedimintd.
+
+  2. Set a Password
+     When starting fedimintd for the first time, each guardian must create a secure password.
+
+  3. Generate Setup Code
+     After entering the password, a unique setup code is generated for each guardian.
+
+  4. Exchange Setup Codes
+     All guardians must share their setup codes with each other (every guardian needs every other guardian’s code).
+
+  5. Distributed Key Generation (DKG)
+     Once all setup codes are entered, the DKG process begins. This is where the federation’s cryptographic keys are jointly created.
+     This step may take some time while all guardians connect.
+
+  6. Federation Complete!
+     Once DKG finishes, your federation is live. Your guardian dashboard will be available for monitoring. You now have a functioning Fedimint!
+
+  # Enabling Lightning Payments
+
+  Fedimint supports Lightning payments via a Lightning Gateway. To use Lightning in your federation, there are two options:
+
+  1. Use an existing Lightning Gateway. [Reach out on Discord to get setup](https://chat.fedimint.org).
+
+  2. Run your own - [see Lightning Gateway docs for more information](https://github.com/fedimint/fedimint/blob/master/docs/gateway.md).
```

Comment:
The plan on umbrelOS eventually is to have "optional dependencies".  e.g if this app exists, I want to integrate with it, but if this app doesn't exist I can still function without it.

It would be cool in the future to have it so that a user can choose to use their own LND instance on their Umbrel as a Gateway and we take care of the initial config (like we do here already for bitcoind).

Reply:
Nice! Excited to tinker and help set that up whenever it's ready.